themes/minimal-gh/minimal-gh.theme.sh

```diff
@@ -0,0 +1,46 @@
+#! bash oh-my-bash.module
+#
+#========================================================================================
+#
+#  ###    ###  ##  ##     ##  ##  ###    ###    ###    ##             ####    ##   ##
+#  ## #  # ##  ##  ####   ##  ##  ## #  # ##   ## ##   ##            ##       ##   ##
+#  ##  ##  ##  ##  ##  ## ##  ##  ##  ##  ##  ##   ##  ##            ##  ###  #######
+#  ##      ##  ##  ##    ###  ##  ##      ##  #######  ##            ##   ##  ##   ##
+#  ##      ##  ##  ##     ##  ##  ##      ##  ##   ##  ######         ####    ##   ##
+#
+#========================================================================================
+#
+# Un tema creado por @AlexGh12 para el proyecto de @ohmybash en CDMX 🇲🇽
+#
+
+function _omb_theme_PROMPT_COMMAND() {
+
+	local TITLEBAR
+	case $TERM in
+	xterm* | screen)
+		TITLEBAR=$'\1\e]0;'$USER@${HOSTNAME%%.*}:${PWD/#$HOME/~}$'\e\\\2' ;;
+	*)
+		TITLEBAR= ;;
+	esac
+
+  local HORA=$(date +%I | sed 's/^0//')
+	local MERIDIANO
+	if [[ HORA -gt 12 ]]; then
```

Comment:
Could you update this? Preceding zeroes can be properly handled by specifying the radix as `10#`. Also, use the arithmetic command for arithmetic comparison.

```suggestion
  local HORA=$(date +%I)
	local MERIDIANO
	if ((10#$HORA > 12)); then
```

Reply:
Also, why `%I` instead of `%H`? The POSIX standard says

> *[POSIX XCU 4 - date](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/date.html)*
> %H
>   Hour (24-hour clock) as a decimal number [00,23].
> %I
>   Hour (12-hour clock) as a decimal number [01,12].

Reply:
You're right, I hadn't noticed the error.
Thanks for the observation and I already uploaded changes